cmd/fpga_plugin/dfl_test.go

```diff
@@ -48,10 +47,12 @@ func TestNewDevicePluginDFL(t *testing.T) {
 	}
 
 	for _, tcase := range tcases {
-		_, err := newDevicePluginDFL("", "", tcase.mode)
-		if tcase.expectedErr && err == nil {
-			t.Error("No error generated when creating Cache with invalid parameters")
-		}
+		t.Run(tcase.mode, func(t *testing.T) {
+			_, err := newDevicePluginDFL("", "", tcase.mode)
+			if tcase.expectedErr && err == nil {
+				t.Error("No error generated when creating Cache with invalid parameters")
+			}
```

Comment:
Could you please add a check for unexpected error?

Reply:
done